lib/preload.js

```diff
@@ -17,10 +17,23 @@
   };
 
   window.onerror = function(msg, file, line, column, err){
-    if (err && msg.indexOf(err.stack) > -1) {
-      err.stack = err.stack + '\n  at ' + file + ':' + line + ':' + column
+    if (err) {
+      console.error((err.stack || err.message || err) + '\n  at ' + file + ':' + line + ':' + column);
+    } else if (msg instanceof Event) {
+      console.error('Unhandled error event at', msg.target, '"' + msg.target + '"')
```

Comment:
why `msg.taget` twice here?

Reply:
The first time it is passed as object, the second time stringified, the reason for this is to easier identify an error once occurred. Depending on the environment the object visualization may help more or less than the toString() info.

Reply:
Can you explain in which case stringifying the object helps?

This is in the browser:

```js
const obj = { oh: 'hi' }
console.log(obj, String(obj))
// => {oh: "hi"} "[object Object]"
```

and this in node:

```js
const obj = { oh: 'hi' }
console.log(obj, String(obj))
// => { oh: 'hi' } [object Object]
```

The `console.error(obj)` should be enough.